_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -22,32 +22,26 @@ $em-base: 16; // deliberately sets bourbon-based em-base (http://bourbon.io/docs
 // #GRID
 // ----------------------------
 // grid - baseline
-$baseline:               20 !default;
-$baseline-vertical:      ($baseline*2) !default;
-$baseline-horizontal:    $baseline !default;
-
-$susy: (
-    columns: 12,
-    container: rem(1170),
-    global-box-sizing: border-box,
-    gutters: .5,
-    math: fluid,
-    output: float,
-    use-custom: (
-        breakpoint: true,
-        background-image: true,
-        background-options: true,
-        box-sizing: false,
-        clearfix: true,
-        rem: true,
-    )
-);
+$baseline:                   20 !default;
+$baseline-vertical:          ($baseline*2) !default;
+$baseline-horizontal:        $baseline !default;
+
+// grid - dimensions
+$grid-max-width:             rem(1170) !default;
+$grid-column-count:          12 !default;
+$grid-gutter-width:          (1/3) !default;  // set to fraction to be proportional to fluid column widths (see - http://susydocs.oddbird.net/en/latest/settings/#gutters)
+$grid-container-position:    center !default;
+$grid-gutter-position:       after !default;
+$grid-manual-gutter-position: inside !default;
+$grid-column-width:          false !default;  // set to false for fluid columns by default (see - http://susydocs.oddbird.net/en/latest/settings/#column-width)
+$grid-debug-color:           rgba(203, 89, 141, 0.33) !default;
 
 // grid - breakpoints
-$bp-small:  320px !default;
-$bp-medium: 768px !default;
-$bp-large:  1024px !default;
-$bp-huge:   1280px !default;
+$bp-screen-sm:                320px !default;
```

Comment:
NOTE: Setting up viewport-dimension based breakpoints.

* We can tailor these once we start with layouts more
* We should add other breakpoints in the future (retina display, media, etc.)